exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/visitor/PrelVisualizerVisitor.java

```diff
@@ -88,10 +88,10 @@ public void endFields() {
     }
 
     public void field(String label, boolean value) {
-      field(label, Boolean.toString(value));
+      append(label, Boolean.toString(value));
     }
 
-    private void field(String label, String value) {
+    private void append(String label, String value) {
```

Comment:
`appendField`?

Reply:
Thanks Bruce for the changes. As Arina pointed out, appendField would be more meaningful in this context.